config/locales/en.yml

```diff
@@ -47,9 +47,15 @@ en:
         title: "Current registered users:"
 
   budgets:
+    attributes:
+      frequencies:
+        weekly: "weekly"
+        fortnightly: "fortnightly"
+        monthly: "monthly"
     defaults:
+      frequency: "Frequency"
       name: "Name"
-      budget_type: "Budget Type"
+      require_budget: "You don't have a budget"
```

Comment:
Maybe change the message to be more informative: "you need to have a budget to be able to use this app"

Reply:
Done in PR, ad-css-styling, sha: 0723be2